cve_bin_tool/cli.py

```diff
@@ -823,37 +824,37 @@ def main(argv=None):
             )
             LOGGER.info(f"Known CVEs in {affected_string}:")
 
-        # Creates an Object for OutputEngine
-        output = OutputEngine(
-            all_cve_data=cve_scanner.all_cve_data,
-            all_cve_version_info=cve_scanner.all_cve_version_info,
-            all_product_data=cve_scanner.all_product_data,
-            scanned_dir=args["directory"],
-            filename=args["output_file"],
-            themes_dir=args["html_theme"],
-            time_of_last_update=cvedb_orig.time_of_last_update,
-            tag=args["tag"],
-            products_with_cve=cve_scanner.products_with_cve,
-            products_without_cve=cve_scanner.products_without_cve,
-            total_files=total_files,
-            is_report=args["report"],
-            append=args["append"],
-            merge_report=merged_reports,
-            affected_versions=args["affected_versions"],
-            exploits=args["exploits"],
-            detailed=args["detailed"],
-            vex_filename=args["vex"],
-        )
+            # Creates an Object for OutputEngine
```

Comment:
I think this indent and the one below were in error and need to be changed back.